c10/core/QuantizedAllocator.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
```

Comment:
ok, so we concluded no separate device and no allocator, right?

Reply:
yeah this is old code, will delete